### PR TITLE
Add script for the launching devfiles testsuite

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
This PR implement next scenario on Centos CI:

* Prepare environment and run minishift
* Deploy CHE
*  Launch devfiles testsuite 
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15517

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
Which issue this PR is aimed to fix?

**Comment by musienko-maxim** [comment]
> Which issue this PR is aimed to fix?
https://github.com/eclipse/che/issues/15517 See in description above

File: tests/.infra/centos-ci/functional_tests_utils.sh [hunk 1]
```diff
@@ -405,4 +405,22 @@ function createIndentityProvider() {
   CHE_OPENSHIFT_PROJECT=eclipse-che
   keycloakPodName=$(oc get pod --namespace=$CHE_OPENSHIFT_PROJECT | grep keycloak | awk '{print $1}')
   /tmp/oc exec $keycloakPodName --namespace=$CHE_OPENSHIFT_PROJECT -- /opt/jboss/keycloak/bin/kcadm.sh create identity-provider/instances -r che -s alias=github -s providerId=github -s enabled=true -s storeToken=true -s addReadTokenRoleOnCreate=true -s 'config.useJwksUrl="true"' -s config.clientId=$CHE_MULTI_USER_GITHUB_CLIENTID_OCP -s config.clientSecret=$CHE_MULTI_USER_GITHUB_SECRET_OCP -s 'config.defaultScope="repo,user,write:public_key"' --no-config --server http://localhost:8080/auth --user admin --password admin --realm master
-}
\ No newline at end of file
+}
+
+function runDevfileTestSuite() {
+  defindCheRoute
```
Comment (rhopp) [inline tests/.infra/centos-ci/functional_tests_utils.sh]: typo? Maybe worth fixing in this PR.

File: tests/.infra/centos-ci/functional_tests_utils.sh [hunk 2]
```diff
@@ -405,4 +405,22 @@ function createIndentityProvider() {
...
+  ### Create directory for report
+  mkdir report
+  REPORT_FOLDER=$(pwd)/report
+  ### Run tests
+  docker run --shm-size=1g --net=host  --ipc=host -v $REPORT_FOLDER:/tmp/e2e/report:Z \
```
Comment (rhopp) [inline tests/.infra/centos-ci/functional_tests_utils.sh:416]: Will the job fail, if the tests fail? Can you confirm this?

File: tests/.infra/centos-ci/functional_tests_utils.sh (hunk 1 above)
Comment (musienko-maxim) [reply tests/.infra/centos-ci/functional_tests_utils.sh]: Looks like it is used just internally the function. I am fixing

File: tests/.infra/centos-ci/functional_tests_utils.sh (hunk 2 above)
Comment (musienko-maxim) [reply tests/.infra/centos-ci/functional_tests_utils.sh:416]: No the  job is not fails we should improve it for happy path and devfiles jobs.

Comment (rhopp) [reply tests/.infra/centos-ci/functional_tests_utils.sh:416]: @musienko-maxim It's up to you if you implement this as part of this PR, or you'll create an issue for implementation of this. Your call.

Comment (musienko-maxim) [reply tests/.infra/centos-ci/functional_tests_utils.sh:416]: i've created issue: https://github.com/eclipse/che/issues/16030

Comment (rhopp) [reply tests/.infra/centos-ci/functional_tests_utils.sh:416]: Thx!

**Review by rhopp — APPROVED** [review]

**Pull request merged**